_dashboards/sc2_ww_dashboard.md

```diff
@@ -13,7 +13,7 @@ links:
 
 ---
 
-<iframe loading="lazy" height="50" width="100%" frameborder="no" scrolling="no" src="https://sc2ww.slhcddcloud.org/wwTime/"><p>Your browser does not support iframes.</p></iframe>
```

Comment:
Please restore this as this is needed to show the time since the last data update. It should sit below the banner.

Reply:
This has been restored